src/main/java/frc/robot/commands/RaiseArmCommand.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.utils.MoPrefs;
+
+public class RaiseArmCommand extends CommandBase {
+    private final ArmSubsystem arms;
+    private final double power;
+    private final BooleanSupplier shouldBeParallel;
+    private boolean wasParallel;
+
+    public RaiseArmCommand(ArmSubsystem arms, double power, BooleanSupplier parallel) {
+        this.arms = arms;
+        this.power = power;
+        this.shouldBeParallel = parallel;
+    }
```

Comment:
The `RaiseArmCommand` isn't requiring the ArmSubsystem? Commands, especially commands that move motors, should always require the subystem they're using, to prevent conflicting motor movements.

The same comment applies to the `RotateWristCommand`.